### PR TITLE
Build: Make main aware 0.8.0

### PR DESCRIPTION
**Pull request by pan3793**
## Summary

Update main branch to aware the latest release 0.8.0

## Checklist
Delete items not relevant to your PR:
- [ ] Unit and integration tests covering the common scenarios were added
- [ ] A human-readable description of the changes was provided to include in CHANGELOG
- [ ] For significant changes, documentation in https://github.com/ClickHouse/clickhouse-docs was updated with further explanations or tutorials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,15 @@ Notes:
 2. Since 0.6.0, HTTP becomes the default protocol.
 3. Since 0.7.0, gRPC is deprecated and not recommended, it may be removed in the future.
 4. Since 0.8.0, gRPC is removed.
+5. Since 0.8.0, the project groupId is rename from `com.github.housepower` to `com.clickhouse.spark`, and
+   the class `xenon.clickhouse.ClickHouseCatalog` is renamed to `com.clickhouse.spark.ClickHouseCatalog`
```
Comment (pan3793) [inline README.md:28]: it's important for users who want to upgrade, should also be mentioned in the release notes

File: docker/compose.yml [hunk 2]
```diff
@@ -152,7 +152,7 @@ services:
       - zookeeper
 
   cloudbeaver:
-    image: pan3793/cloudbeaver:20240223
+    image: dbeaver/cloudbeaver:24.1
```
Comment (pan3793) [inline docker/compose.yml:155]: switch to official dbeaver image as i already pushed the custom changes into upstream

File: docker/image/scc-metastore.Dockerfile [hunk 3]
```diff
@@ -24,7 +24,7 @@ ENV HIVE_HOME=/opt/hive
 ENV HIVE_CONF_DIR=/etc/hive/conf
 
 RUN set -x && \
-    wget -q ${APACHE_MIRROR}/hive/hive-${HIVE_VERSION}/apache-hive-${HIVE_VERSION}-bin.tar.gz && \
+    wget -q https://archive.apache.org/dist/hive/hive-${HIVE_VERSION}/apache-hive-${HIVE_VERSION}-bin.tar.gz && \
```
Comment (pan3793) [inline docker/image/scc-metastore.Dockerfile:27]: we have to get the hive 2.3.9 from apache archive site as the hive community removed such a version from the dln site.

File: docs/quick_start/01_get_the_library.md [hunk 4]
```diff
@@ -22,16 +22,16 @@ The name pattern of binary jar is
 ```
 clickhouse-spark-runtime-${spark_binary_version}_${scala_binary_version}-${version}.jar
 ```
-you can find all available released jars under [Maven Central Repository](https://repo1.maven.org/maven2/com/github/housepower)
-and all daily build SNAPSHOT jars under [Sonatype OSS Snapshots Repository](https://oss.sonatype.org/content/repositories/snapshots/com/github/housepower/).
+you can find all available released jars under [Maven Central Repository](https://repo1.maven.org/maven2/com/clickhouse/spark)
```
Comment (pan3793) [inline docs/quick_start/01_get_the_library.md:25]: i checked the jars are there

File: version.txt [hunk 5]
```diff
@@ -1 +1 @@
-0.8.0-SNAPSHOT
+0.9.0-SNAPSHOT
```
Comment (pan3793) [inline version.txt:1]: the main branch should go 0.9.0-SNAPSHOT now, and the branch-0.8 should go 0.8.1-SNAPSHOT now

**Comment by pan3793** [comment]
ping @mzitnik

**Comment by mzitnik** [comment]
@pan3793 Thanks

**Review by mzitnik — APPROVED** [review]

**Pull request merged**